test/trafficgen/kapinger.yaml

```diff
@@ -47,7 +47,7 @@ spec:
       serviceAccountName: kapinger-sa
       containers:
         - name: kapinger
-          image: ghcr.io/microsoft/retina/kapinger:latest
+          image: acnpublic.azurecr.io/kapinger:20241009.5
```

Comment:
Do we want to push the image to `ghcr.io`?

Reply:
workflow is ready, just missing creds
https://github.com/microsoft/retina/blob/main/.github/workflows/kapinger.yaml#L25